nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/DocumentModelComparator.java

```diff
@@ -106,7 +106,7 @@ public int compare(DocumentModel doc1, DocumentModel doc2) {
             DataModel d2 = doc2.getDataModel(schemaName);
             for (Entry<String, String> e : orderBy.entrySet()) {
                 final String fieldName = e.getKey();
-                final boolean asc = ORDER_ASC.equals(e.getValue());
+                final boolean asc = ORDER_ASC.equalsIgnoreCase(e.getValue());
```

Comment:
For APIs I don't see the point in ignoring case. The developer should use whatever case is specified in the API (here uppercase I guess, or better, the constants — maybe we could add a `ORDER_DESC` to be clear).

So -1 on this change as it is. I'll comment on the ticket.

Reply:
ok thanks